project_template/models.py

```diff
@@ -292,6 +292,8 @@ class OwnedIncomeFromInvestmentsTable(models.Model):
 # Tabel Venituri investitii - actual row information
 class OwnedIncomeFromInvestmentsTableEntry(CommonIncomeFields):
     table = models.ForeignKey(OwnedIncomeFromInvestmentsTable, on_delete=models.CASCADE, null=True)
+    receiver_of_goods = models.ForeignKey(Person, on_delete=models.CASCADE, null=True)
```

Comment:
The mapping between the political figure who is the owner and the beneficiary of all the objects inside the asset declaration and the asset declaration itself is already implemented. 

The implementation works as follows: a `Declaration` model has a `Politician` as a foreign key ("One politician can have one or more declarations"), a `*Table` model (for example `OwnedInvestmentsTable`) has a `declaration` model as a foreign key ("One declaration can have multiple tables, among which one is a `OwnedInvestmentsTable` table).

Reply:
So it also doesn't make any sense to have name and surname in the corresponding form, right?

Reply:
So the name and surname retrieved from this form should go through this mapping, and not have a new one here.